config.in

```diff
@@ -4,6 +4,11 @@
 #
 # Read `man 5 sway` for a complete reference.
 
+### Session setup
+#
+# Power off screen on idle and raise lockscreen
+exec swayidle -x 'swaylock -c 000000' timeout 300 exec 'swaylock -c 000000' timeout 600 exec 'swaymsg "output * dpms off"' resume exec 'swaymsg "output * dpms on"'
```

Comment:
Can you split this onto several lines with \ and comment it out by default?

Reply:
Done